lib/buildArtefacts/artifacts-public-access.ts

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
Looks like a typo in the file name `buildArtefacts` -> `buildArtifacts`. Might want to change all imports too after this change

Reply:
fixed the naming